lib/core/include/irods/irods_logger.hpp

```diff
@@ -485,13 +493,48 @@ namespace irods::experimental::log
             /// \since 4.3.0
             template <typename... Args>
             auto operator()(fmt::format_string<Args...> _format, Args&&... _args) const -> void
+#if __clang__ == 0 || __clang_major__ > 15
+                // Clang 15 and later (and GCC 11 and later) consider this overload ambiguous
+                // with the std::invocable overload. This constraint sorts that out.
+                requires (sizeof...(Args) != 1 || !std::is_invocable_v<std::tuple_element_t<0, std::tuple<Args...>>>)
+#elif __clang_major__ == 15
+                // Clang 15 can't parse the above statement properly, so we have special overloads for 15.
+                // This overload handles the 1-arguemnt case.
```

Comment:
Sorry, I just saw this... Feel free to ignore. :p
```suggestion
                // This overload handles the 1-argument case.
```

Reply:
Fixed